### DISCUSSION TITLE
How to Restrict input to max length_

### DISCUSSION THREAD

**Discussion by SpeedyVV** [danvick/flutter_form_builder#822] [Q&A] [answered]
I have a field for ID number which has 9 digits.

Is there a way to stop user from inputting more numbers after the 9th number is entered?

Thanks

**Comment by subhasundardass** [comment, marked as answer]
FormBuilderTextField(
    maxLength: 9,
    decoration: InputDecoration(
    labelText: 'ID',
    ),
    keyboardType: TextInputType.number,
)
